pkg/server/registry/apigroups/acorn/regions/strategy.go

```diff
@@ -0,0 +1,59 @@
+package regions
+
+import (
+	"context"
+
+	apiv1 "github.com/acorn-io/acorn/pkg/apis/api.acorn.io/v1"
+	v1 "github.com/acorn-io/acorn/pkg/apis/internal.acorn.io/v1"
+	"github.com/acorn-io/mink/pkg/types"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apiserver/pkg/storage"
+)
+
+type strategy struct {
+	startTime metav1.Time
+}
+
+func (s *strategy) Get(_ context.Context, _, name string) (types.Object, error) {
+	if name != "local" {
+		return nil, apierrors.NewNotFound(schema.GroupResource{
+			Group:    apiv1.SchemeGroupVersion.Group,
+			Resource: "regions",
+		}, name)
+	}
+
+	return &apiv1.Region{
```

Comment:
This is cool. I assume we don't persist region, we just return a local region.

Reply:
Yep, that's right!